README.md

```diff
@@ -165,63 +165,29 @@ npm run build
 ```
 Then, run the server:
 ```bash
-npm run start:http
-```
-
-### SSE Server
-
-This runs the server as a Server-Sent Events (SSE) service on port 3001.
-
-**Development (without building):**
-```bash
-npm run serve:sse
-```
-
-**Production:**
-First, build the project:
-```bash
-npm run build
-```
-Then, run the server:
-```bash
-npm run start:sse
+npm run start:web
 ```
 
 ## Docker
 
-This project includes two Dockerfiles to run the Swarm MCP server as a containerized service, either via HTTP or Server-Sent Events (SSE).
+This project includes a Dockerfile to run the Swarm MCP server as a containerized service, with both HTTP and SSE transports.
 
-- `Dockerfile`: The default Dockerfile for the HTTP server (port 3000).
-- `Dockerfile.sse`: Builds an image for the SSE server (port 3001).
+- `Dockerfile`: Builds a single image for the server, which runs on port 3000.
 
-### Building the Docker Images
+### Building the Docker Image
 
-To build the Docker images, run the following commands from the project root:
+To build the Docker image, run the following command from the project root:
 
-**HTTP Server:**
 ```bash
-docker build -t swarm-mcp-http .
+docker build -t swarm-mcp .
 ```
 
-**SSE Server:**
-```bash
-docker build -t swarm-mcp-sse -f Dockerfile.sse .
-```
-
-### Running the Docker Containers
-
-**HTTP Server:**
-To run the HTTP server, use the `docker run` command. The server is exposed on port `3000`.
-
-```bash
-docker run --name swarm-mcp-http -p 3000:3000 swarm-mcp-http
-```
+### Running the Docker Container
 
-**SSE Server:**
-To run the SSE server, use the `docker run` command. The server is exposed on port `3001`.
+To run the server, use the `docker run` command. The container exposes port `3000` for both HTTP and SSE.
```

Comment:
it would be enough to just mention this (Streamable) HTTP and SSE support only once.